ssm_tools/ssm_session_cli.py

```diff
@@ -57,21 +57,26 @@ def parse_args(argv):
     if bool(args.INSTANCE) + bool(args.list) != 1:
         parser.error("Specify either INSTANCE or --list")
 
-    return args
+    return args, extras
 
-def start_session(instance_id, profile=None, region=None):
+def start_session(instance_id, extras, profile=None, region=None):
     extra_args = ""
     if profile:
         extra_args += f"--profile {profile} "
     if region:
         extra_args += f"--region {region} "
-    command = f'aws {extra_args} ssm start-session --target {instance_id}'
+
+    parameters = ""
+    for extra in extras:
+        parameters += f" {extra}"
+
+    command = f'aws {extra_args} ssm start-session --target {instance_id}{parameters}'
```

Comment:
Should there be a space in `{instance_id}{parameters}`?

Reply:
No because we prefix a space in the for loop. That way if there are no parameters, we won’t add an extra space.

Reply:
True. So maybe `parameters = " ".join(extras)` in that case?

Reply:
And by the way an extra space wouldn't really matter while making the string more _aesthetically pleasing_ :)

Reply:
Sounds good to me 👍